src/app/home/home.component.html

```diff
@@ -86,7 +86,7 @@ <h1>Reserve a Day-Pass</h1>
         <div class="col"></div>
         <div class="col-4">
             <h2 class="py-2">
-                Select your Park
+                Select a Park
```

Comment:
noticed this banner should probably have the rounded edges too

Reply:
bdc7339